docs/sources/flow/reference/components/otelcol.exporter.loadbalancing.md

```diff
@@ -0,0 +1,263 @@
+* `"service"`: exports spans based on their service name. This is useful when using processors like the span metrics, 
+so all spans for each service are sent to consistent collector instances for metric collection. 
+Otherwise, metrics for the same services would be sent to different collectors, making aggregations inaccurate.
+* `"traceID"`: exports spans based on their traceID.
+
+## Blocks
+
+The following blocks are supported inside the definition of
+`otelcol.exporter.loadbalancing`:
+
+Hierarchy | Block | Description | Required
+--------- | ----- | ----------- | --------
+resolver | [resolver][] | Configures discovering the endpoints to export to. | yes
+resolver > static | [static][] | Static list of endpoints to export to. | no
+resolver > dns | [dns][] | DNS-sourced list of endpoints to export to. | no
+protocol | [protocol][] | Protocol settings. Only OTLP is supported at the moment. | no
+protocol > otlp | [otlp][] | Configures an OTLP exporter. | no
+protocol > otlp > client | [client][] | Configures the exporter gRPC client. | no
+protocol > otlp > client > tls | [tls][] | Configures TLS for the gRPC client. | no
+protocol > otlp > client > keepalive | [keepalive][] | Configures keepalive settings for the gRPC client. | no
+protocol > otlp > queue | [queue][] | Configures batching of data before sending. | no
+protocol > otlp > retry | [retry][] | Configures retry mechanism for failed requests. | no
+
+The `>` symbol indicates deeper levels of nesting. For example, `resolver > static`
+refers to a `static` block defined inside a `resolver` block.
+
+[resolver]: #resolver-block
+[static]: #static-block
+[dns]: #dns-block
+[protocol]: #protocol-block
+[otlp]: #otlp-block
+[client]: #client-block
+[tls]: #tls-block
+[keepalive]: #keepalive-block
+[queue]: #queue-block
+[retry]: #retry-block
+
+### resolver block
+
+The `resolver` block configures how to retrieve the endpoint to which this exporter will send data.
+
+Inside the `resolver` block, either the [dns][] block or the [static][] block 
+should be specified. If both `dns` and `static` are specified, `dns` takes precedence.
+
+### static block
+
+The `static` block configures a list of endpoints which this exporter will send data to.
+
+The following arguments are supported:
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`hostnames` | `list(string)` | List of endpoints to export to. |  | yes
+
+### dns block
+
+The `dns` block periodically resolves an IP address via the DNS `hostname` attribute. This IP address 
+and the port specified via the `port` attribute will then be used by the gRPC exporter 
+as the endpoint to which to send data to.
+
+The following arguments are supported:
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`hostname` | `string`   | DNS hostname to resolve. |  | yes
+`port`     | `string`   | Port to be used with the resolved IP address. | `"4317"` | no
```

Comment:
Should this be expanded out, because at first glance it looks like the hostname port. `Port to be used with the IP addresses returned from the DNS hostname`?